### DISCUSSION TITLE
it's safe to not disable NvStrapsReBar for deep clean my pc ?

### DISCUSSION THREAD

**Discussion by wandaww** [terminatorul/NvStrapsReBar#83] [Q&A]
hi dev, i want to deep clean my pc which is requiring to re-assemble the component but nothing change to the pc component just to clean it. it is safe to not disable the NvStrapsRebar ? i worry because the bios can be reset it sometimes even the CMOS battery not plugged out when i do deep clean my pc. (or maybe just my overclock setting reset idk, just want to ask.)

**Comment by terminatorul** [comment]
In the general case the answer is no. You should disable ReBAR if you make hardware changes or UEFI Setup changes. Even if you reassemble everything just the way it was, internal settings in UEFI would still change.

**Comment by terminatorul** [comment]
If v0.4-rc1 works for you, it will disable ReBAR automatically on UEFI setup chagnes, and you only need to re-enable it afterwards
